packages/editor/src/components/post-featured-image/panel.js

```diff
@@ -15,7 +15,7 @@ import PostFeaturedImageCheck from './check';
 
 const PANEL_NAME = 'featured-image';
 
-function FeaturedImage() {
+export function PrivatePostFeaturedImagePanel( { renderPanelBody } ) {
```

Comment:
What are your thoughts on phasing out the existing component or providing a public API for the new one?

Reply:
I think we can revisit later with the move of more components and while refining the design. This is the 'safer' option for now and it's harmless.

Reply:
One idea could be exposing the `withPanelBody` property and set it to `true` by default — the existing uses will remain the same but we'd use the public component for the new use cases (instead of requiring a private one).

Reply:
I know, and I guess this prop is not bad to be public API right away. I'll update.